Formula/textidote.rb

```diff
@@ -0,0 +1,55 @@
+class Textidote < Formula
+  desc "Spelling, grammar and style checking on LaTeX documents"
+  homepage "https://sylvainhalle.github.io/textidote"
+  url "https://github.com/sylvainhalle/textidote/archive/refs/tags/v0.8.3.tar.gz"
+  sha256 "8c55d6f6f35d51fb5b84e7dcc86a4041e06b3f92d6a919023dc332ba2effd584"
+  license "GPL-3.0-only"
+  head "https://github.com/sylvainhalle/textidote.git", branch: "master"
+
+  depends_on "ant" => :build
+  depends_on "openjdk"
+
+  def install
+    # Build the JAR
+    system "ant", "download-deps"
+    system "ant", "-Dbuild.targetjdk=1.7"
```

Comment:
since you are using `openjdk`, why not setting this to `17` rather than `1.7`?

Looks like they are using default jdk as 1.6?
https://github.com/sylvainhalle/textidote/blob/25c223f439dbfafcee923e4b3c4d55d8ada454f0/build.xml#L44

Reply:
Yes, they used `1.6` as default. However, this did not work on my machine ("Target option 6 is no longer supported. Use 7 or later.").

I chose to set to `1.7` to ensure compatibility, but if openjdk guarantees JDK17 to be available on all machines, then I'm OK.
I just tested it on my machine, and the build works with `17`.

Reply:
```suggestion
    system "ant", "-Dbuild.targetjdk=17"
```